etc/tendrl/node-agent/node-agent.conf.yaml.sample

```diff
@@ -5,3 +5,4 @@ tendrl_ansible_exec_file: $HOME/.tendrl/node-agent/ansible_exec
 log_cfg_path: /etc/tendrl/node-agent/node-agent_logging.yaml
 log_level: DEBUG
 package_source_type: rpm
+logging_socket_path: "/etc/tendrl/.node_agent_socket"
```

Comment:
this default path is not supposed to be in /etc,  application sockets can be store in /tmp/node-agent/.node_agent.sock